web/templates/tooltip_templates.hbs

```diff
@@ -107,14 +107,14 @@
     </div>
 </template>
 <template id="all-message-tooltip-template">
-    <div class="views-tooltip-container" data-view-code="all_messages">
```

Comment:
Could this `data-view-code` refactor be put in a prep commit or PR? I'm not sure what it's about, but it seems like it could be done earlier to help simplify review.